frontend/src/Api/General/LanguageComponentConstants.elm

```diff
@@ -135,8 +135,8 @@ german =
     , month = "mm"
     , printJournal = "Journal drucken"
     , printNominalAccounts = "Sachkonten drucken"
-    , accountCategories = germanAccountCategories
-    , accountTypes = germanAccountTypes
+    , accountCategories = makeCategoryList germanAccountCategories
+    , accountTypes = makeTypeList germanAccountTypes
```

Comment:
`makeCategoryList` seems very natural in this context. But why do language components need to know about categories (referring to `categoryIds`)? If the categories are required only in another context, it might be better to add another function `categories : AccountType -> List CategoryNumber` and use this function, whenever one needs the list of the category ids.